GraphBulkImporter/Program.cs

```diff
@@ -0,0 +1,194 @@
+            var token = tokenSource.Token;
+
+            BulkImportResponse vResponse = null;
+            BulkImportResponse eResponse = null;
+
+            try
+            {
+                vResponse = await graphbulkExecutor.BulkImportAsync(
+                        Utils.GenerateVertices<Store>(numberOfDocumentsToGenerate),
+                        enableUpsert: true,
+                        disableAutomaticIdGeneration: true,
+                        maxConcurrencyPerPartitionKeyRange: null,
+                        maxInMemorySortingBatchSize: null,
+                        cancellationToken: token);
+
+                eResponse = await graphbulkExecutor.BulkImportAsync(
+                        Utils.GenerateEdges(numberOfDocumentsToGenerate),
+                        enableUpsert: true,
+                        disableAutomaticIdGeneration: true,
+                        maxConcurrencyPerPartitionKeyRange: null,
+                        maxInMemorySortingBatchSize: null,
+                        cancellationToken: token);
+            }
+            catch (DocumentClientException de)
+            {
+                Trace.TraceError("Document client exception: {0}", de);
+            }
+            catch (Exception e)
+            {
+                Trace.TraceError("Exception: {0}", e);
+            }
+
+            Console.WriteLine("\nSummary for batch");
+            Console.WriteLine("--------------------------------------------------------------------- ");
+            Console.WriteLine(
+                "Inserted {0} graph elements ({1} vertices, {2} edges) @ {3} writes/s, {4} RU/s in {5} sec)",
+                vResponse.NumberOfDocumentsImported + eResponse.NumberOfDocumentsImported,
+                vResponse.NumberOfDocumentsImported,
+                eResponse.NumberOfDocumentsImported,
+                Math.Round(
+                    (vResponse.NumberOfDocumentsImported) /
+                    (vResponse.TotalTimeTaken.TotalSeconds + eResponse.TotalTimeTaken.TotalSeconds)),
+                Math.Round(
+                    (vResponse.TotalRequestUnitsConsumed + eResponse.TotalRequestUnitsConsumed) /
+                    (vResponse.TotalTimeTaken.TotalSeconds + eResponse.TotalTimeTaken.TotalSeconds)),
+                vResponse.TotalTimeTaken.TotalSeconds + eResponse.TotalTimeTaken.TotalSeconds);
+            Console.WriteLine(
+                "Average RU consumption per insert: {0}",
+                (vResponse.TotalRequestUnitsConsumed + eResponse.TotalRequestUnitsConsumed) /
+                (vResponse.NumberOfDocumentsImported + eResponse.NumberOfDocumentsImported));
+            Console.WriteLine("---------------------------------------------------------------------\n ");
+
+            if (vResponse.BadInputDocuments.Count > 0 || eResponse.BadInputDocuments.Count > 0)
+            {
+                using (System.IO.StreamWriter file = new System.IO.StreamWriter(@".\BadVertices.txt", true))
```

Comment:
`.\` would be implied even if you just write `@"BadVertices.txt"` and is cross-platform in case we can make this into dotnet core